src/test/java/org/kohsuke/github/GHRepositoryTest.java

```diff
@@ -1594,4 +1594,34 @@ public void testRepoActionVariable() throws Exception {
         GHRepositoryVariable variable = repository.getRepoVariable("myvar");
         assertThat(variable.getValue(), is("this is my var value"));
     }
+
+    /**
+     * Test create repo action variable.
+     *
+     * @throws IOException
+     *             the exception
+     */
+    @Test
+    public void testCreateRepoActionVariable() throws IOException {
+        GHRepository repository = getRepository();
+        GHRepositoryVariable newVariable = new GHRepositoryVariable();
+        newVariable.setName("mynewvariable");
+        newVariable.setValue("mynewvalue");
+        repository.createRepoVariable(newVariable);
```

Comment:
Since both `name` and `value` are required the method signature should probably look like: 
```suggestion
        repository.createRepoVariable("mynewvariable", "mynewvalue");
```